prep-javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,35 @@
+const number1 = 6,
+  number2 = 4,
+  product = 4 * 6;
```

Comment:
Totally valid, however you won't see people creating variables like this in JavaScript that much. The generally accepted way is to break them up into their own line for readability purposes. Functionality still the same, and you are free to use whichever makes the most sense to you, but wanted to highlight that you won't see this often